### PR TITLE
Various tweaks and fixes

### PR DESCRIPTION
**Pull request by GAsplund**
# Key Features:
- Don't try to create or overwrite a file if it exists
- Modify Dockerfile for better caching and media storage
- Add safety nets for various errors
- Prevent duplicate banner creations
- "Select an item"-placeholders for drop-downs when adding Gamma groups and selecting banner groups
- Sticky position for header and footer (contact card)
- Remove CID from database schema and replace with GammaUserId (see #53)
- Create a base database migration
- Move server-side action files from /app directory to common directory
	- Not sure if this should be done, but I feel like they belong together similarly to how services do

# Known Issues:
- This should remove issues rather than fix them lol

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -39,6 +40,10 @@ COPY --from=builder --chown=nextjs:nodejs /app/.next/static ./.next/static
 COPY --chown=nextjs:nodejs next-logger.config.js /app/
 COPY --chown=nextjs:nodejs pino-pretty-transport.js /app/
 
+# Create media directory
+RUN mkdir -p $MEDIA_PATH
+RUN chown -R nextjs:nodejs $MEDIA_PATH
+
```
Comment (Oscariremma) [inline Dockerfile:46]: This is fine but when actually running the docker image in production a volume will probably be mounted in this location which "overwrites" the folder created here. That is fine but that means that the chown will not apply and the folder will probably be owned by root as default which needs to be handled when the application starts somehow.

**Review by Oscariremma — COMMENTED** [review]
I am willing to approve the PR in it's current state but see the comment on the docker file

File: Dockerfile (hunk 1 above)
Comment (GAsplund) [reply Dockerfile:46]: Hmm, that might be difficult as the container doesn't run as the root user. Will have to do some research on this.

Comment (Oscariremma) [reply Dockerfile:46]: After testing it seams to work

**Review by Oscariremma — APPROVED** [review]

**Pull request merged**